examples/mixture/plot_concentration_prior.py

```diff
@@ -0,0 +1,130 @@
+"""
+========================================================================
+Concentration Prior Type Analysis of Variation Bayesian Gaussian Mixture
+========================================================================
+
+This example plots the ellipsoids obtained from a toy dataset (mixture of three
+Gaussians) fitted by the ``BayesianGaussianMixture`` class models with a
+Dirichlet process distribution
+(``weight_concencration_prior_type='dirichlet_distribution'``) and a Dirichlet
+process prior (``weight_concencration_prior_type='dirichlet_process'``). On
+each figure, we plot the results for three different values of the weight
+concentration prior.
+
+The ``BayesianGaussianMixture`` can adapt its number of mixture automatically.
+The parameter ``weight_concentration_prior`` has a direct link with the
+resulting number of components. Specifying higher values more often leads to
+uniformly- sized mixture components, while specifying smaller values will lead
```

Comment:
Please remove the reference to uniformly distributed components which is not true for the DP prior.

Add a note at the end of the paragraph stating that the DD prior will favor more uniformly weighted components.